### PR TITLE
Language sections of onboarding page. KKB-474

### PR DESCRIPTION
**Pull request by rasben**
The onboarding administration page now has two sections - english and danish.

Both sections are displayed on the same page, per the design
https://projects.invisionapp.com/share/QE10RDRRPSTZ#/screens

Screenshot 2021-04-28 at 08 44 05

### CODE REVIEW

File: kkb_registration/kkb_registration.admin.inc [hunk 1]
```diff
@@ -32,170 +67,234 @@ function kkb_registration_admin_form($form, &$form_state) {
...
-        // add a bunch of logic to increment the number for an edgecase.
-        $anchor_cleaned .= '0';
+        continue;
       }
 
-      $values[$info_key]['anchor'] = $anchor_cleaned;
-      $anchors[] = $anchor_cleaned;
+      if (!empty($info['anchor'])) {
+        // By using clean_css, we make sure that the anchor is valid for URLs.
+        $anchor_cleaned = drupal_html_id($info['anchor']);
```
Comment (kasperg) [inline kkb_registration/kkb_registration.admin.inc:279]: `clean_css` is not relevant anymore.

File: kkb_registration/kkb_registration.admin.inc [hunk 2]
```diff
@@ -32,170 +67,234 @@ function kkb_registration_admin_form($form, &$form_state) {
...
+
+        // Making sure anchor IDs are unique.
+        if (in_array($anchor_cleaned, $anchors)) {
+          // Yes, this mean it can result in #anchorlink0000 but it's silly to
+          // add a bunch of logic to increment the number for an edgecase.
+          $anchor_cleaned .= '0';
+        }
+
+        $infos[$info_key]['anchor'] = $anchor_cleaned;
+        $anchors[] = $anchor_cleaned;
```
Comment (kasperg) [inline kkb_registration/kkb_registration.admin.inc:291]: All this should be handled by `drupal_html_id()`.

File: kkb_registration/kkb_registration.module [hunk 3]
```diff
@@ -35,34 +35,39 @@ function kkb_registration_theme() {
...
-  $image = NULL;
-  $image_fid = variable_get('kkb_registration_image', NULL);
+  // We have several language-grouped sections of content.
+  // The sections is showed on the same page though, as Ding2 doesn't have
+  // a split-language setup.
+  // Instead, we add a lang="LANG_KEY" to make Google & co understand.
+  $languages = [
+    'da' => variable_get('kkb_registration_da', NULL),
+    'en' => variable_get('kkb_registration_en', NULL),
+  ];
```
Comment (kasperg) [inline kkb_registration/kkb_registration.module:55]: :+1: for explaining.

**Review by kasperg — APPROVED** [review]
:+1: but remember to check out the unique id generation.

**Comment by rasben** [comment]
I'm fixing the three comments in the next PR:
https://github.com/kdb/kkb-ddb-modules/pull/86

**Pull request merged**